scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1458,21 +1485,22 @@ def formPipelineJson(pipelineId, dbPipelineJson, stageJson):
         raise e
 
 
-
 def postingGateJsonSQL(pipelineJson):
     try:
         pipeline_id = pipelineJson['id']
-        update_data = json.dumps(pipelineJson), isd_admin_username, pipeline_id	
-        mysqlcursor_front50 = spindb_front50.cursor()
+        update_data = json.dumps(pipelineJson), isd_admin_username, pipeline_id        
         sql = "UPDATE pipelines SET body = %s, last_modified_by = %s WHERE id = %s"        
-        mysqlcursor_front50.execute(sql, update_data)
+        if spin_db_type == 'sql':
+           mysqlcursor_front50 = spindb_front50_sql.cursor()
+           mysqlcursor_front50.execute(sql, update_data)
+        elif spin_db_type == 'postgres':
```

Comment:
@sudhakaropsmx please relook at the cusrors. Why do we need this type of handling.
Couldn't you just use a common cursor for postgres and sql db.... based on the input param we can create open cursor (front50 and orca) on either postgres connection or sql connection (we know it will be either of them, not both)

Reply:
mysql is different from postgres

Reply:
I am not convinced. May be you didn't understand the ask here...let's review in a call